doc/deployment_guide.md

```diff
@@ -0,0 +1,806 @@
+Oct 28 16:37:15 mdip-gatekeeper start-node[706734]: tbtc-mediator-1  | ]
+...
+Oct 28 16:37:17 mdip-gatekeeper start-node[706734]: tbtc-mediator-1  | {
+Oct 28 16:37:17 mdip-gatekeeper start-node[706734]: tbtc-mediator-1  |     "hex": "0200000000010172471d5072fe4b3cca7b1280d12e3ae419bf021358f7fbde00c07bd9faf886a50100000000fdffffff0200000000000000003e6a3c6469643a746573743a7a3376384175616258426d4e4d50784c4454596e4655724c5173667835526a414479343348506a355758666f384e7546764a57c2450000000000001600141b3544273e9a63b04cba431d086563a780f84c15024730440220068107db0675ed3fe6c32b6a4f5e2523c26ddcdaed812fc9be5858eb3cbcb9ea022005a9a9d7927424112e87719835aa8f7f17df6b1373c32d1764f5528d085a17730121027e15ea510377a03013154c70e66a9851db5591e428c370f51e12f6bbd3a7016400000000",
+Oct 28 16:37:17 mdip-gatekeeper start-node[706734]: tbtc-mediator-1  |     "complete": true
+Oct 28 16:37:17 mdip-gatekeeper start-node[706734]: tbtc-mediator-1  | }
+Oct 28 16:37:17 mdip-gatekeeper start-node[706734]: tbtc-mediator-1  | 0.00017858000000000001
+Oct 28 16:37:17 mdip-gatekeeper start-node[706734]: tbtc-node-1      | 2024-10-28T19:43:17.461639Z [mdip] AddToWallet 4b52dfa95d7b9472147b2821a33c59cdd52bf1444b6c0feed4e7c8c0d9a6f798  new InMempool
+Oct 28 16:37:17 mdip-gatekeeper start-node[706734]: tbtc-mediator-1  | Transaction broadcast with txid: 4b52dfa95d7b9472147b2821a33c59cdd52bf1444b6c0feed4e7c8c0d9a6f798
+Oct 28 16:37:18 mdip-gatekeeper start-node[706734]: gatekeeper-1     | POST /api/v1/queue/TBTC/clear 200 650.769 ms - 4
+Oct 28 16:37:18 mdip-gatekeeper start-node[706734]: tbtc-mediator-1  | export loop waiting 1 minute(s)...
+```
+
+## Secure Hosting an MDIP Node with nginx
+
+nginx [https://nginx.org/](https://nginx.org/) is a popular web server and reverse proxy available for most operating systems.
+
+nginx can be used to control and throttle access to server ports and hosted services like MDIP. The configuration files presented here are for educational purposes. Securing a server for online operations involves more than protecting a few ports; this document limits itself to MDIP-specific ports and services.
+
+### Important MDIP Ports and Endpoints
+
+A full MDIP node launched using the provided `kc/docker-compose.yml` configuration will only expose 2 new service ports on the Host machine:
+
+```docker
+keychainmdip/keymaster   0.0.0.0:4226->4226/tcp   kc-keymaster-1
+keychainmdip/gatekeeper  0.0.0.0:4224->4224/tcp   kc-gatekeeper-1
+```
+
+When using Docker as an operating environment, other ports are used within the Docker network to mediate with 3rd party systems.
+
+```docker
+mongo:latest             27017/tcp kc-mongodb-1
+bitcoin-core:v27.99.0    48332/tcp kc-tbtc-node-1
+feathercoind             19337/tcp kc-tftc-node-1
+```
+
+None of the ports listed above should ever be directly addressable from the Internet. It is recommended that a firewall be setup to prevent access to these services.
```

Comment:
FYI Firewalling is always a good idea, but since these ports aren't being mapped to the host they _should_ be inaccessible outside of the internal docker network.